PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -81,40 +75,26 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
     correctionFile = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2016_UltraLegacy_preVFP_RunFineEtaR9Gain"),
 )
 
-(run2_egamma_2016 & tracker_apv_vfp30_2016).toModify(calibratedPatPhotonsNano,
+(run2_egamma_2016 & tracker_apv_vfp30_2016).toModify(
+    calibratedPatPhotonsNano,
     correctionFile = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2016_UltraLegacy_preVFP_RunFineEtaR9Gain")
 )
 
-(run2_egamma_2016 & ~tracker_apv_vfp30_2016).toModify(calibratedPatPhotonsNano,
+(run2_egamma_2016 & ~tracker_apv_vfp30_2016).toModify(
+    calibratedPatPhotonsNano,
     correctionFile = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2016_UltraLegacy_postVFP_RunFineEtaR9Gain"),
 )
 
-run2_egamma_2017.toModify(calibratedPatPhotonsNano,
+run2_egamma_2017.toModify(
+    calibratedPatPhotonsNano,
     correctionFile = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2017_24Feb2020_runEtaR9Gain_v2")
 )
 
-run2_egamma_2018.toModify(calibratedPatPhotonsNano,
+run2_egamma_2018.toModify(
+    calibratedPatPhotonsNano,
     correctionFile = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2018_29Sep2020_RunFineEtaR9Gain")
 )
 
-run2_nanoAOD_102Xv1.toModify(calibratedPatPhotonsNano,
-    correctionFile = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2018_Step2Closure_CoarseEtaR9Gain_v2")
-)
-
-run2_nanoAOD_94XMiniAODv1.toModify(calibratedPatPhotonsNano,
-    correctionFile = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2017_17Nov2017_v1_ele_unc")
-)
-
-run2_nanoAOD_94XMiniAODv2.toModify(calibratedPatPhotonsNano,
-    correctionFile = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2017_17Nov2017_v1_ele_unc")
-)
-
-
-run2_miniAOD_80XLegacy.toModify(calibratedPatPhotonsNano,
-                                correctionFile = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Legacy2016_07Aug2017_FineEtaR9_v3_ele_unc")
-)
-
-
 slimmedPhotonsWithUserData = cms.EDProducer("PATPhotonUserDataEmbedder",
```

Comment:
Contrary to the electrons, it seems that the V1 cutBased ID is still used in the output table, but we might want to remove it... perhaps @cms-sw/egamma-pog-l2 can comment?